packages/grid_client/tests/modules/gateways.test.ts

```diff
@@ -162,78 +213,100 @@ test("TC1237 - Gateways: Expose a VM Over Gateway", async () => {
   expect(result[0].capacity["cpu"]).toBe(cpu);
   expect(result[0].capacity["memory"]).toBe(memory);
   expect(result[0].planetary).toBeDefined();
-  expect(result[0].publicIP).toBeNull();
+  expect(result[0].myceliumIP).toBeDefined();
+  expect(result[0].publicIP["ip"]).toBeDefined();
+  expect(result[0].publicIP["ip6"]).toBeDefined();
   expect(result[0].description).toBe(description);
 
-  const backends = ["http://[" + result[0].planetary + "]:8000"];
+  //--------------------Planetary--------------------
+  let backends = ["http://[" + result[0].planetary + "]:8000"];
   log(backends);
 
   //Name Gateway Model
   const gw: GatewayNameModel = {
-    name: name,
+    name: name + "planetary",
     node_id: gatewayNodeId,
     tls_passthrough: tlsPassthrough,
     backends: backends,
   };
 
-  const gatewayRes = await gridClient.gateway.deploy_name(gw);
-  log(gatewayRes);
+  await testGateway(gw, name);
 
```

Comment:
```suggestion
  await testGateway(gw, gw.name);
```

Reply:
Good point, I got the name from gateway itself.